core/src/main/java/org/jboss/galleon/layout/FeaturePackLayout.java

```diff
@@ -157,6 +159,19 @@ public ConfigLayerSpec loadConfigLayerSpec(String model, String name) throws Pro
         }
     }
 
+    public ConfigModel loadModel(String model) throws ProvisioningException {
```

Comment:
Given this method is not used in the PR, does this mean it's meant for use elsewhere? I ask, because it's a compatibility issue requiring any mutually inclusive upgrades.

Reply:
@jamezp , yes It is used in glow project that depends on galleon 5.2.0.Final that includes it.